src/bun.js/webcore/blob/ReadFile.zig

```diff
@@ -34,15 +34,14 @@ pub fn NewReadFileHandler(comptime Function: anytype) type {
                 .result => |result| {
                     const bytes = result.buf;
                     if (blob.size > 0)
-                        blob.size = @min(@as(u32, @truncate(bytes.len)), blob.size);
-                    const value = Function(&blob, globalThis, bytes, .temporary);
+                        blob.size = @min(@as(Blob.SizeType, @truncate(bytes.len)), blob.size);
```

Comment:
if `blob.size` is less than `bytes.len` we just report the `blob.size` but we still pass the full bytes (with is more than the blob.size) to the `Function(b, g, by, .temporary)` should we slice it to match the blob.size?

Reply:
we want the full memory allocation so it can be freed